app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Models\WaitingRoom;
+use Illuminate\Http\Request;
+
+class WaitingRoomsController extends ApiController
+{
+    public function calculateSplit(Request $request)
```

Comment:
I think @weerd makes good points on the "split" between API vs "web" controllers.